torchci/pages/flakytest.tsx

```diff
@@ -29,8 +30,9 @@ export default function Page() {
         <div>Loading...</div>
       ) : (
         (data.flakyTests as FlakyTestData[]).map((test) => {
+          const samples = data.flakySamples[`${test.name}, ${test.suite}`];
```

Comment:
can we import this method from flakytest.ts. Not sure if everyone will know how to encode these keys.